src/components/Notifications/RuleSimple/RuleSimple.js

```diff
@@ -238,6 +348,36 @@ export default class RuleSimple extends React.Component {
     }
   }
 
+  getDefaultJoinColumnAndDisplayNameAndJoinColumnsIndices = (queryResult) => {
```

Comment:
Is this not the same function in JoinColimnSelectionTable? If so it should be moved to utils function

Reply:
Sure,  you are right, it's better to move it to Utility librarys